ansible/README.md

```diff
@@ -0,0 +1,60 @@
+# Provisioning RPC node and Graph Node
+
+## Requirements
+- [Ansible](https://www.ansible.com/)
+- [Vagrant](https://www.vagrantup.com/downloads)
+- A virtualization env such as: VirtualBox, VMware Fusion, or Hyper-V
+- Server with Debian 11
+
+## Description
+Ansible manages the installation of any software component using different [roles](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#roles).
+
+### group_vars
+It contains all the variables used in every role. Please change accordingly to install a sokol or xdai node. 
+It also allows to add users for the rpc nethermind node under `allowed_users`.
+
+### roles
+- **basic-setup**: installs required packages and enable unattended security updates. 
+- **nethermind**: installs a customised helm chart for nethermind node using the xdai network and adc hoc k8s resources required to secure the node. 
+    - Version  [v1.12.7](https://github.com/NethermindEth/nethermind/releases/tag/1.12.7)
+    - This node has security enabled and not everyone can connect to it. To be added please write to circlesubi@
```

Comment:
Are we supposed to write our email here?

Reply:
oh yes